### PR TITLE
Confirm civicrm coding standards

### PR DESCRIPTION
**Pull request by lucky091588**

### CODE REVIEW

File: CRM/Cplabels/Form/Search/Cplabels/Client.php [hunk 1]
```diff
@@ -125,18 +125,15 @@ public function buildForm(&$form) {
...
+    $form->addRule('team', E::ts('Field "Team" is required'), 'required');
+    $form->addRule('diagnosis', E::ts('Field "Diagnosis" is required'), 'required');
   }
 
   /**
    * Get a list of summary data points
    *
-   * @return mixed; NULL or array with keys:
-   *  - summary: string
-   *  - total: numeric
```
Comment (twomice) [inline CRM/Cplabels/Form/Search/Cplabels/Client.php:137]: I'd like to keep the @return documentation in these lines. The lint error for these two lines is "Return comment indentation must be 3 spaces, found 2 spaces". So I think you can fix this lint error by simply adjusting the space at the front of these two lines (136 and 137).

File: CRM/Cplabels/Form/Search/Cplabels/Client.php [hunk 2]
```diff
@@ -125,18 +125,15 @@ public function buildForm(&$form) {
...
-    $form->addRule('team', E::ts('Field "Team" is required'),'required');
-    $form->addRule('diagnosis', E::ts('Field "Diagnosis" is required'),'required');
+    $form->addRule('team', E::ts('Field "Team" is required'), 'required');
+    $form->addRule('diagnosis', E::ts('Field "Diagnosis" is required'), 'required');
   }
 
   /**
    * Get a list of summary data points
    *
-   * @return mixed; NULL or array with keys:
```
Comment (twomice) [inline CRM/Cplabels/Form/Search/Cplabels/Client.php:135]: I'd like to keep the @return documentation in these lines. The lint error here is ` Return type "mixed; NULL or array with keys:" must not contain spaces`. So I think you can fix this just by moving `NULL or array with keys:` to the next line.

File: CRM/Cplabels/Form/Search/Cplabels/CommonGround.php [hunk 3]
```diff
@@ -18,18 +19,15 @@ public function buildForm(&$form) {
...
     // This search has no editable criteria.
-    $form->addElement('static', 'statichtml', '', '<p class="status alert">'. E::ts('This search has no editable criteria. Please click "Search".') . '</p>');
+    $form->addElement('static', 'statichtml', '', '<p class="status alert">' . E::ts('This search has no editable criteria. Please click "Search".') . '</p>');
     $form->assign('elements', array('statichtml'));
   }
 
   /**
    * Get a list of summary data points
    *
-   * @return mixed; NULL or array with keys:
```
Comment (twomice) [inline CRM/Cplabels/Form/Search/Cplabels/CommonGround.php:28]: I'd like to keep the `@return` documentation here. Please see my comments above about the similar topic. Thanks!

File: CRM/Cplabels/Form/Search/Cplabels/Volunteer.php [hunk 4]
```diff
@@ -58,18 +59,15 @@ public function buildForm(&$form) {
...
+    $form->addRule('serve_type', E::ts('Field "Service type" is required'), 'required');
   }
 
   /**
    * Get a list of summary data points
    *
-   * @return mixed; NULL or array with keys:
-   *  - summary: string
-   *  - total: numeric
    */
```
Comment (twomice) [inline CRM/Cplabels/Form/Search/Cplabels/Volunteer.php:73]: I'd like to keep the @return documentation here. Please see my comments above about the similar topic. Thanks!

**Review by twomice — COMMENTED** [review]
Looks good, @lucky091588 Just a few comments to better address lint errors around `@return` docblock comments. Thanks for your help.

File: CRM/Cplabels/Form/Search/Cplabels/Volunteer.php (hunk 4 above)
Comment (lucky091588) [reply CRM/Cplabels/Form/Search/Cplabels/Volunteer.php:73]: Fixed

File: CRM/Cplabels/Form/Search/Cplabels/CommonGround.php (hunk 3 above)
Comment (lucky091588) [reply CRM/Cplabels/Form/Search/Cplabels/CommonGround.php:28]: Fixed

File: CRM/Cplabels/Form/Search/Cplabels/Client.php (hunk 2 above)
Comment (lucky091588) [reply CRM/Cplabels/Form/Search/Cplabels/Client.php:135]: Fixed

File: CRM/Cplabels/Form/Search/Cplabels/Client.php (hunk 1 above)
Comment (lucky091588) [reply CRM/Cplabels/Form/Search/Cplabels/Client.php:137]: Fixed

**Comment by twomice** [comment]
Merged, thanks @lucky091588 !

**Pull request merged**